echopype/echodata/zarr_combine.py

```diff
@@ -633,6 +634,7 @@ def write_to_file(
                 compute=True,
                 storage_options=storage_options,
                 synchronizer=zarr.ThreadSynchronizer(),
+                consolidated=consolidated,
```

Comment:
You should not add consolidated here. The metadata has already been created at this point and you are recomputing it.

Reply:
Aren't you writing the merged dataset into a new zarr file here?

Reply:
Within this function we are just filling in values to a zarr store that has already been consolidated. A quick way to make this determination is to see if `region` is an input of `to_zarr`. If `region` is an input, then the metadata should already be constructed.